### ISSUE TITLE
document about minimal MAF before annotation

### ISSUE THREAD

**Issue by jjgao** [cBioPortal/cbioportal#4853]
The MAF format in the (current documentation)(https://cbioportal.readthedocs.io/en/latest/File-Formats.html) is too complex and not accurate. For example, I think we should always require the genomic changes columns.

![image](https://user-images.githubusercontent.com/840895/45890957-4d075080-bd92-11e8-9b22-bd75bf6f73c2.png)

Maybe we improve it by:

- Document the minimal MAF that is required for annotating through Genome Nexus (instead of importing into the database)
- Document other important columns that are used in the portal, e.g. read counts
- Point Genome Nexus for annotation

@inodb @pieterlukasse @sandertan

**Comment by sandertan** [comment]
Sounds good. We could link to the official MAF documentation, https://docs.gdc.cancer.gov/Data/File_Formats/MAF_Format/ for the explanation of column names, and focus this documentat on the relevant columns for cBioPortal functionality.

What are the columns required for genome nexus annotation? And will Genome Nexus annotation always be done, or only when specific fields are missing?

**Comment by jjgao** [comment]
@sandertan Good idea to point to the GDC MAF document and focus our doc on cBioPortal functionality.

GN only requires 5 columns of genomic changes as minimal input. (we should probably have `NCBI_Build` as well -- currently only support GRCh37/hg19)

A MAF should be ran through GN annotation at least once to normalize the annotations.

**Comment by inodb** [comment]
I think there might be some flag to not annotate, so CMO can force their annotations over genome nexus, but that might only be part of pipelines' code. @angelicaochoa do u know?

**Comment by angelicaochoa** [comment]
@inodb MAFs will be annotated with genome nexus on the fly if column `HGVSp_Short` is not present in the file. The CMO does not pre-annotate MAFs so unless someone does this manually then they will always undergo annotation as part of import pipeline

**Comment by pieterlukasse** [comment]
@jjgao does this mean we should start recommending using GN instead of VCF2MAF/VEP for the annotation step? If GN can be assumed to be a "given" at some point (i.e. we make it a dependency for cBioPortal), then I think the mutation data format could be indeed simplified since the extra annotation will happen either at the time of import or on the fly in the cBioPortal platform itself.

**Comment by jjgao** [comment]
@pieterlukasse @sandertan 

CMO means Center for Molecular Oncology -- it's our department. We have a lot of internal data coming from CMO.

Currently annotation is done either when the data is prepared or imported.

Our pipelines have switched to GN, but VCF2MAF is perfectly fine at this moment for annotating MAF. At some point, we should recommend using GN, but maybe after we refactor the GN annotation? @inodb 

Also to clarify, when I said "A MAF should be ran through GN annotation at least once to normalize the annotations", I mean "MAFs should go through the same annotation process, e.g. same canonical isoforms, for an instance of cBioPortal."

**Comment by pieterlukasse** [comment]
@jjgao thanks for the clarification. Coming back to the first point you mentioned in this ticket:
 - Document the minimal MAF that is required for annotating through Genome Nexus (instead of importing into the database)
I don't think would be correct, unless the GN step can be assumed to always run during import step.
